apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -167,42 +166,54 @@ function CustomEventDialogs(props: CustomEventDialogProps) {
                     </IconButton>
                 </Tooltip>
             )}
-            <Dialog open={open} onClose={handleClose} maxWidth={'lg'}>
+            <Dialog open={open} onClose={handleClose} maxWidth={'xs'}>
+                <DialogTitle id="form-dialog-title" style={{ marginBottom: -10 }}>
```

Comment:
```suggestion
                <DialogTitle id="form-dialog-title">
```
**edit: the issue+suggestion below are still good practice, but read the comment replying to this one as for why we opt to remove the custom styling altogether**

issue: using marginBottom to override another property (in this case `paddingBottom`) is generally bad practice

suggestion: whenever possible, prefer to identify if you can directly target the style that you would otherwise want to (in this case) "cancel out"

----

nit: Also, when working with MUI v5 (denoted by the import `@mui/material`), use `sx` when styling components

Reply:
issue: looking at the code below which adds `marginTop` to the element directly below, removing `marginBottom` from `DialogTitle` and `marginTop` from the event name input element achieves the same effect.